src/addons/__tests__/renderSubtreeIntoContainer.js

```diff
@@ -54,7 +54,7 @@ describe('renderSubtreeIntoContainer', function() {
     });
 
     ReactTestUtils.renderIntoDocument(<Parent />);
-    expect(portal.firstChild.innerHTML).toBe('bar');
+    // TODO: expect(portal.firstChild.innerHTML).toBe('bar'); after parent-context lands.
```

Comment:
In the test here, can you assign `this.component = <Component />` in render so it gets owner-based context, then make sure there's no context warning? (And that there is a warning if the context doesn't match?)